spring-petclinic/pom.xml

```diff
@@ -108,83 +110,75 @@
         </pluginRepository>
     </pluginRepositories>
 
-    <properties>
+	<properties>
+
+		<!-- Explicitly declaring the source encoding eliminates the following message: -->
+		<!-- [WARNING] Using platform encoding (UTF-8 actually) to copy filtered 
+			resources, i.e. build is platform dependent! -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
 		<!-- JBoss dependency versions -->
-		<jboss.maven.plugin.version>7.4.Final</jboss.maven.plugin.version>
+		<version.wildfly.maven.plugin>1.0.2.Final</version.wildfly.maven.plugin>
         <version.jboss.bom.eap>7.0.0-build-7</version.jboss.bom.eap>
+		<version.jboss.spec.javaee.7.0>1.0.3.Final</version.jboss.spec.javaee.7.0>
+
+		<!-- Generic properties -->
+		<java.version>1.7</java.version>
```

Comment:
This seems to be an unused property and conflicts with the definition about 1.8 bellow. Can we remove it or you just left it to be closed to the original version ?